### ISSUE TITLE
Typo in Segmentation Chapter (16.1 page 171)

### ISSUE THREAD

**Issue by burakgorduk** [remzi-arpacidusseau/ostep-typos#99]
Looks like the order of words is incorrect for _size_ and _segment_:

`The size segment here is exactly the same as
the bounds register introduced previously`

should be

`The segment size here is exactly the same as
the bounds register introduced previously`

Unless I'm misunderstanding it.

**Comment by remzi-arpacidusseau** [comment]
Good catch! The actual fix here is "The size register" (!) Thanks! You'll be ack'd in the preface eventually.

**Comment by burakgorduk** [comment]
Thank **you** for this amazing book. It has been influential in helping me get to where I am today.
